File: csharp/DVSClient/DVSClient/address/lookup/LookupLocalityItem.cs
using DVSClient.Server.Address.Lookup;

namespace DVSClient.Address.Lookup
{
    public class LookupLocalityItem
    {
        public string Name { get; }
        public string Code { get; }
        public string Description { get; }

        public LookupLocalityItem(RestApiAddressLookupLocalityItem response)
        {
            Name = response.Name??"";
            Code = response.Code??"";
            Description = response.Description??"";
        }
    }
}

File: typescript/src/address/lookup/lookupLocalityItem.ts
export type LookupLocalityItem = {
    name?: string;
    code?: string;
    description?: string;
};

export function restApiResponseToLookupLocalityItem(response: RestApiAddressLookupLocalityItem): LookupLocalityItem {
    return {
        name: response.name??"",
        code: response.code??"",
        description: response.description??""
    }
}